Better secrets handling for Pantheon (#810)
Gizra/drupal-starter: README.md

## Two-factor Authentication (TFA)

TFA is disabled by default. Edit the [Pantheon-specific settings](https://github.com/Gizra/drupal-starter/blob/main/web/sites/default/settings.pantheon.php#L96) to activate it.
The default settings under `/admin/config/people/tfa` define "Skip Validation" is 1. That is,
when a privileged user will login, they must enable their TFA. Otherwise, on a second
login, they will already be blocked. A site admin may reset their validation tries
under the `/admin/people` page.
The TFA method that is enabled is one that uses Google authenticator (or similar).

You should set the TFA secret using:
```bash
ddev terminus secret:site:set gizra-drupal-starter tfa_key $(openssl rand -base64 32) --type=runtime --scope=web,user
```

> If you need environment-based override, you can do the following:

Reviewer: What's the diff between this command and the one above?

Author: The `.qa` suffix - specifying where the value should be applied.

Reviewer: ```suggestion
If you need to override the secret on a specific Pantheon environment:
```
